RogueMaster/flipperzero-firmware-wPlugins
# Added ability to create "loader apps" for use on Main Menu

# What's new

### Loader App
- It's a dummy app created for the purpose of attaching an application.fam file to use with Main Menu.
- I added a `link` variable to the application.fam file. It's used to specify the FAP file location needed for loading

### Loader.c
- Changed loader.c to incorporate the application.fam `link` variable change which lets any FAP be on the Main Menu with the use of a "loader app"
- This removes the hard coded app loading (Clock/iButton/U2F) and creates it dynamically

## Note
- If no `link` variable is specified in application.fam, it's automatically set to NULL which is then ignored by loader.c when launching the app.

# Verification 

- Create a dummy app using clock_loader, ibutton_loader, or u2f_loader as an example.
- Ensure the proper application.fam details are filled in, with the location of the FAP file needed for loading.
- Ensure loader_app is included in Main/applications.fam file
- Build
- Test if app shows up on Main Menu and loads FAP correctly.
- Also test if other apps still work correctly.

# Checklist (For Reviewer)
